### PR TITLE
blog intro

### PR DESCRIPTION
**Pull request by emmagrob**
it is not finished, please feel free to add comments in the meantime. thanks!

### CODE REVIEW

File: wix-blog-backend/wix-blog-backend/guides/blog-intro.md [hunk 1]
```diff
@@ -4,6 +4,19 @@ title: Introduction
 
 # Introduction
 
+Wix Blog allows site owners to create and manage blogs. They can create and manage posts, categories and tags on their blog page. Site contributers can also create and manage blogs given the appropriate roles and permissions.
```
Comment (roysha-wix) [inline wix-blog-backend/wix-blog-backend/guides/blog-intro.md:7]: - Who does 'they' and 'their' refer to in the second sentence?

File: wix-blog-backend/wix-blog-backend/guides/blog-intro.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ title: Introduction
 
 # Introduction
 
+Wix Blog allows site owners to create and manage blogs. They can create and manage posts, categories and tags on their blog page. Site contributers can also create and manage blogs given the appropriate roles and permissions. 
+
+There are 3 types of blog roles: 
+> - **Blog Editors** can fully manage the blog, but not other areas of a site.
+> - **Blog Writers** can create and publish their own posts on a blog.
+> - **Guest writer** can only create posts. Their posts must be reviewed and published by a site owner or blog editor.
```
Comment (roysha-wix) [inline wix-blog-backend/wix-blog-backend/guides/blog-intro.md:12]: Their posts must be reviewed ---> Posts written by guest writers must be reviewed

**Review by roysha-wix — COMMENTED** [review]
Best intro ever

File: wix-blog-backend/wix-blog-backend/guides/blog-intro.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Before you begin
+
+Before using any `wixBlogBackend` functions, [add Wix Blog](https://support.wix.com/en/article/wix-blog-adding-and-setting-up-your-blog) to your site. 
+
+To use the wix-blog-backend API, import `wixBlogBackend` from the `wix-blog-backend` module. 
+
+```javascript
+import wixBlogBackend from 'wix-blog-backend';
+```
```
Comment (imnotashrimp) [inline wix-blog-backend/wix-blog-backend/guides/blog-intro.md:19]: Move this to above line 11 (Before you begin)

**Review by imnotashrimp — COMMENTED** [review]
The REST and Velo intros should be closely aligned, so my comments on your REST intro are applicable here as well.

**Pull request closed without merging**